src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java

```diff
@@ -443,36 +415,6 @@ protected boolean testFileSystem() {
         return true;
     }
 
-    protected boolean testFreeSpace() {
-        if (DIRS.length == 0)
-            return true;
```

Comment:
Why deleting this method?

Reply:
It is nowhere used in our application

Reply:
It is used - in `onRequestPermissionsResult` and `onCreate`.

It should optimized instead.

Reply:
You mean the method `testFileSystem` not this one.

Reply:
You are right. 
The `testFreeSpace` was deactivated in 70c1a930c1eb23c00b16f55fe346752f47b2ecc8.

Please check if now som strings are obsolete.

Reply:
Found and removed